src/command/j.jl

```diff
@@ -25,7 +25,7 @@ end
 function commander(c::Client, m::Message, ::Val{:julia_doc})
     # @info "julia_commander called"
     # @info "Message content" m.content m.author.username m.author.discriminator
-    startswith(m.content, COMMAND_PREFIX * "j") || return
+    startswith(m.content, COMMAND_PREFIX * "j ") || startswith(m.content, COMMAND_PREFIX * "j? ") || m.content == COMMAND_PREFIX * "j" || return
```

Comment:
Can you break the long line? 

BlueStyle suggests 92 characters.
https://github.com/invenia/BlueStyle#synopsis

These formatting changes can be automated by the editor
https://github.com/invenia/BlueStyle#editor-configuration

Reply:
Yeah, that is a good thing. I just did that. In this line and in a few others.